### ISSUE TITLE
Add <findAidDesc> as replacement of <filedesc>

### ISSUE THREAD

**Issue by kerstarno** [SAA-SDT/eas-schemas#27]
## Creator of issue

1. Kerstin Arnold
2. EAD team lead, TS-EAS
3. @kerstarno
4. kerstin.arnold@archivesportaleurope.net

## The issue relates to

- [ ] EAC-CPF schema issue
- [ ] EAC-CPF Tag Library issue
- [x] EAD schema issue
- [x] EAD Tag Library issue
- [ ] Schema issue
- [ ] Tag Library issue
- [ ] Suggestions for all schemas
- [ ] Suggestions for all Tag Libraries
- [ ] Other

## Wanted change/feature

* Text: Add `<findAidDesc>` as an optional and repeatable sub-element of `<ead>`. While optional (and hence usually listed after the required sibling elements `<control>` and `<archDesc>`), `<findAidDesc>` should sit between `<control>` and `<archDesc>`.

**Update (4 January 2024):
Following the EAD team's meeting on 15 December and representing the latest status of decisions, the details of this issue have been updated. The current development version of the schema might still represent what was outlined initially,  i.e. further changes might be necessary and require another round of testing.**

**_Note for working on and testing this issue: When the schema changes are done in development branch, please mark this by ticking the box on the higher level (printed in bold); once the changes have been tested successfully please mark this by ticking the box on the lower levels._**

- [x] **Apply the following content model for `<findAidDesc>`**
  - [x] Add `<agent>` as a sub-element of `<findAidDesc>` (see extended definition in #32; see also #58)
  - [x] Add `<citedRange>` as a sub-element of `<findAidDesc>` (as defined in `<source>`, see #32)
  - [x] Add `<date>` as a sub-element of `<findAidDesc>` (as defined as a shared element, see #4)
  - [ ] ~~Add `<div>` as a sub-element of `<findAidDesc>` (reintroduced from deprecated EAD3, see definition below)~~
  - [x] Add `<formattingExtension>` as a sub-element of `<findAidDesc>` (see #62) 
  - [x] Add `<place>` as a sub-element of `<findAidDesc>` (as defined as a shared element, see #42)
  - [x] Add `<title>` as a sub-element of `<findAidDesc>` (as defined in `<controlAccess>`, see #13)
  - [x] Require at least one out of these sub-elements to be present within `<findAidDesc>` 
  - [x] Have all of these sub-elements as repeatable within `<findAidDesc>`
- [x] **Include the following optional attributes in `<findAidDesc>`**
  - [x] `@audience`, `@id`, and `@target`
  - [x] `@languageOfElement` and `@scriptOfElement`
  - [ ] ~~`@encodingAnalog` (see #37)~~
  - [x] `@conventionDeclarationReference`, `@maintenanceEventReference`, and `@sourceReference`
  - [x] `@href`, `@linkRole`, and `@linkTitle` 
  - [x] An option to include attributes from any other namespace
- [ ] **~~Define `<div>` as sub-element of `<findAidDesc>` according to its current definition within EAD3 undeprecated~~**
  - [ ] ~~Add `<head>` as sub-element of `<div>` (optional, not repeatable)~~
  - [ ] ~~Add `<blockQuote>`, `<chronList>`, `<div>`, `<list>`, `<p>`, and `<table>` as sub-elements of `<div>`~~
  - [ ] ~~Require at least one out of these sub-elements (all but `<head>`) to be present within `<div>` (_different from EAD3 undeprecated_)~~
  - [ ] ~~Have all of these sub-elements (all  but `<head>`) as repeatable within `<div>`~~
  - [ ] ~~Add `@audience`, `@id`, and `@target`~~
  - [ ] ~~Add `@languageOfElement` and `@scriptOfElement`~~
  - [ ] ~~Add an option to include attributes from any other namespace~~

**Comment by kerstarno** [comment]
For reference, this is how a transformation from EAD 2002 to EAD 4.0 with regard to `<findAidDesc>` could look like (elements with * are mandatory):

**EAD 2002**
```
<ead>
  <eadheader>*
    <eadid/>*
    <filedesc>*
      <titlestmt>*
        <titleproper/>*
        <subtitle/>
        <author/>
        <sponsor/>
      </titlestmt>
      <editionstmt>
        <edition/>
        <p/>
      </editionstmt>
      <publicationstmt>
        <publisher/>
        <date/>
        <address/>
        <num/>
        <p/>
      </publicationstmt>
      <seriesstmt>
        <titleproper/>
        <num/>
        <p/>
      </seriesstmt>
      <notestmt>
        <note>
          <address/>
          <blockquote/>
          <chronlist/>
          <list/>
          <note/>
          <p/>
          <table/>
        </note>
      </notestmt>
    </filedesc>
  </eadheader>
  <frontmatter>
    <titlepage>
      <address/>
      <blockquote/>
      <chronlist/>
      <list/>
      <note/>
      <p/>
      <table/>
      <author/>
      <date/>
      <edition/>
      <num/>
      <publisher/>
      <bibseries/>
      <sponsor/>
      <titleproper/>
      <subtitle/>
    </titlepage>
    <div>
      <address/>
      <head/>
      <blockquote/>  
      <chronlist/>
      <list/>
      <note/>
      <p/>
      <table/>
      <div/>
    </div>
  </frontmatter>    
  </archdesc>*
</ead>
```

**EAD 4.0**
```
<ead>
  <control maintenanceStatus*=”...”>*
    <recordId/>*
    <maintenanceAgency/>*
    <maintenanceHistory/>*
  </control>  
  <findAidDesc href=”...” etc.>
    <title/>
    <title localType=”subtitle”/>
    <agent agentRole=”author”/>
    <agent agentRole=”sponsor”/>
    <!-- <editionstmt><edition> and <editionstmt><p> would be converted to <eventDescription> -->
    <agent agentRole=”publisher”/>
    <date/>
    <place>
      <address/>
    </place>
    <title localType=”publicationNumber”/>
    <div>
      <p/>
    </div>
    <title localType=”serialTitle”/>
    <title localType=”serialNumber”/>
    <div>
      <p/>
    </div>
    <place>
      <address/>
    </place>
    <div>
      <blockquote/>
      <chronlist/>
      <list/>
      <div/> 
      <p/>
      <table/>
    </div>
    <place>
      <address/>
    </place>
    <div>
      <blockquote/>
      <chronlist/>
      <list/>
      <div/>
      <p/>
      <table/>
    </div>
    <agent agentRole=”author”/>
    <date/>
    <!-- <frontmatter><edition> would be converted to <eventDescription> -->
    <title localType=”number”/>
    <agent agentRole=”publisher/>
    <!-- <bibseries> was deprecated in EAD3 and will be removed in EAD 4.0 -->
    <agent agentRole=”sponsor”/>
    <title/>
    <title localType=”subtitle”/>
    <place>
      <address/>
    </place>
    <div>
      <head/>
      <blockquote/>
      <chronlist/>
      <list/>
      <div/>
      <p/>
      <table/>
      <div/>
    </div>
  </findAidDesc>
  <archdesc/>*
</ead>
```

**Comment by kerstarno** [comment]
For reference, this is how a transformation from EAD3 to EAD 4.0 with regard to `<findAidDesc>` could look like (elements with * are mandatory):

**EAD3**
```
<ead>
  <control>*
    <maintenanceStatus/>*
    <recordid/>*
    <maintenanceagency/>*
    <maintenancehistory/>*
    <filedesc>*
      <titlestmt>*
        <titleproper/>*
        <subtitle/>
        <author/>
        <sponsor/>
      </titlestmt>
      <editionstmt>
        <edition/>
        <p/>
      </editionstmt>
      <publicationstmt>
        <publisher/>
        <date/>
        <address/>
        <num/>
        <p/>
      </publicationstmt>
      <seriesstmt>
        <titleproper/>
        <num/>
        <p/>
      </seriesstmt>
      <notestmt>
        <controlnote>
          <blockquote/>
          <chronlist/>
          <list/>
          <p/>
          <table/>
        </controlnote>
      </notestmt>
    </filedesc>
  </control>  
  <frontmatter> (deprecated) 
    <titlepage> (deprecated) 
      <blockquote/>
      <chronlist/>
      <list/>
      <p/>
      <table/>
      <author/>
      <date/>
      <edition/>
      <num/>
      <publisher/>
      <bibseries/> (deprecated) 
      <sponsor/>
      <titleproper/>
      <subtitle/>
    </titlepage>
    <div> (deprecated) 
      <head/>
      <blockquote/>  
      <chronlist/>
      <list/>
      <p/>
      <table/>
      <div/>
    </div>
  </frontmatter> 
  </archdesc>*
</ead>
```

**EAD 4.0**
```
<ead>
  <control maintenanceStatus*=”...”>*
    <recordId/>*
    <maintenanceAgency/>*
    <maintenanceHistory/>*
  </control>  
  <findAidDesc href=”...” etc.>
    <title/>
    <title localType=”subtitle”/>
    <agent agentRole=”author”/>
    <agent agentRole=”sponsor”/>
    <!-- <editionstmt><edition> and <editionstmt><p> would be converted to <eventDescription> -->
    <agent agentRole=”publisher”/>
    <date/>
    <place>
      <address/>
    </place>
    <title localType=”publicationNumber”/>
    <div>
      <p/>
    </div>
    <title localType=”serialTitle”/>
    <title localType=”serialNumber”/>
    <div>
      <p/>
    </div>
    <div>
      <blockquote/>
      <chronlist/>
      <list/>
      <p/>
      <table/>
    </div>
    <div>
      <blockquote/>
      <chronlist/>
      <list/>
      <div/>
      <p/>
      <table/>
    </div>
    <agent agentRole=”author”/>
    <date/>
    <!-- <frontmatter><edition> would be converted to <eventDescription> -->
    <title localType=”number”/>
    <agent agentRole=”publisher/>
    <!-- <bibseries> was deprecated in EAD3 and will be removed in EAD 4.0 -->
    <agent agentRole=”sponsor”/>
    <title/>
    <title localType=”subtitle”/>
    <div>
      <head/>
      <blockquote/>
      <chronlist/>
      <list/>
      <p/>
      <table/>
      <div/>
    </div>
  </findAidDesc>
  <archdesc/>*
</ead>
```

**Comment by kerstarno** [comment]
Note: depending on the review of #53, `<findAidDesc>` might need to be named `<findingAidDescription>`.

**Comment by kerstarno** [comment]
@fordmadox - Not sure, if the editing of existing comments results in a notification email, so adding a comment here as well to note that this issue (description and encoding examples) has been updated re the inclusion of `<formattingExtension>` instead of `<div>`.

**Comment by fordmadox** [comment]
What about `title`?  Is that the only element from controlAccess that isn't being redefined or removed?

**Comment by kerstarno** [comment]
Yes, indeed, `<title>` is kept, though only used in this context here and not anymore in other contexts where it was used together with the other access elements. 

There's still be a few changes to `<title>` though compared to how it is defined on EAD3. All of these should already be covered by other issues anyway, so the following list is mainly for completeness: 

- The default changes applied to most elements, i.e. 
  - Adapt `@audience`
  - Remove `@altrender`
  - Add `@target`
  - Rename `@lang` and `@script` to `@languageOfElement` and `@scriptOfElement`
  - Add optional attributes `@conventionDeclarationReference`, `@maintenanceEventReference`, `@sourceReference`
  - Add option to include attributes from any other namespace
- The changes with regard to the vocabulary attributes (as mentioned in #13), i.e. 
  - Replace `@identifier` with `@valueURI`
  - Replace `@source` with `@vocabularySource`
  - Add `@vocabularySourceURI`
  - Remove `@rules`
  - Remove `@normal`
  - Remove `@relator`
- Some other general changes, i.e.
  - Remove `@encodinganalog`
  - Apply camelCase spelling to `@localType`
  - Add `@localTypeDeclarationReference`
  - Replace `@render` with `@style`

The sub-element `<part>` is adapted to the way it's defined in EAC-CPF 2.0.

**Comment by kerstarno** [comment]
Tested with the XSD and the RNG. There are a few changes still required:

- `<findAidDesc>` should be repeatable
- `<findAidDesc>` should not include `@base`, but it should include the linking attributes (`@href`, `@linkRole`, and `@linkTitle`)
- `<formattingExtension>` in `<findAidDesc>` should be treated in the same way as the other sub-elements (it seems that the reference being "element.formattingExtension.optional" resulting in `<formattingExtension>` being marked additionally as optional here).

I have created a pull request for these changes (#78). Once this pull request has been merged, this issue will need retesting.
